app/src/main/java/com/android/solvit/shared/ui/navigation/NavigationActions.kt

```diff
@@ -51,6 +52,8 @@ object Screen {
   const val REVIEW_SCREEN = "Review Screen"
   const val EDIT_PREFERENCES = "Edit Preferences"
   const val NOTIFICATIONS = "Notifications Screen"
+  const val AI_SOLVER_WELCOME_SCREEN = "Ai Get Started Screen"
+  const val AI_SOLVER_CHAT_SCREEN = "AI Solver Chat Screen"
```

Comment:
Im not sure that the optimal option would be adding a screen for each UI screen as this overloads the concept of segmenting main screens. Check for State-Managed Child Screens like the one I implemented in the Job Dashboard of the provider

Reply:
You think ! I don't really get the difference (as in the bootcamp for these kind of screens I defined a route and two screens)

Reply:
We can discuss about it in next stand up